go/pkg/pwapi/api_team-send-invite.go

```diff
@@ -55,23 +55,17 @@ func (svc *service) TeamSendInvite(ctx context.Context, in *TeamSendInvite_Input
 		return nil, errcode.ErrNotTeamOwner.Wrap(err)
 	}
 
-	// check that the invited user is not already in this season
-	var existCheck pwdb.TeamMember
+	// check if invited user already has a team in this season
+	var existingSeasonMembership pwdb.TeamMember
 	err = svc.db.
-		Where(pwdb.TeamMember{
-			UserID: invitedUserID,
-		}).
-		Where(&pwdb.Season{
-			ID: team.SeasonID,
-		}).
-		Joins("JOIN team ON team_member.team_id = team.id").
-		Joins("JOIN season ON team.season_id = season.id").
-		First(&existCheck).
+		Model(pwdb.TeamMember{}).
+		Joins("JOIN team on team.id = team_member.team_id AND team.season_id = ? AND team.deletion_status = ?", team.SeasonID, pwdb.DeletionStatus_Active).
```

Comment:
```suggestion
		Joins("JOIN team on team.id = team_member.team_id").
		Where("team.season_id = ?", team.SeasonID).
		Where("team.deletion_status = ?", team.SeasonID, pwdb.DeletionStatus_Active).
```

Reply:
btw I've a doubt if it works to use the alternative `Where(Team{SeasonID: team.SeasonID})` format or if gorm will skip the field prefix

but anyway, please, split the where as I suggested